docs/source/operations/upgrade_fabricnetwork.md

```diff
@@ -0,0 +1,26 @@
+<a name = "upgrading-fabric"></a>
+# Upgrading Hyperledger Fabric version
+
+- [Pre-requisites](#pre_req)
+- [Modifying image versions](#modify_image_version)
+
+<a name = "pre_req"></a>
+## Pre-requisites
+Hyperledger Fabric image versions, which are compatible with the target fabric version need to be known. Here's the list of the images for which the versions should be known:  
+kafka image-- hyperledger/fabric-kafka:`targer_version`  
+zookeeper image-- hyperledger/fabric-zookeeper:`target_version`  
+couchDB image-- hyperledger/fabric-couchdb:`target_version`  
+
+---
+**NOTE:** The upgrade based solely on the base image version change will work only if the fabric upgrade requires the change of image versions only
```

Comment:
"This change only upgrades the docker images, any other configuration changes is not covered by this guide. Please refer to Fabric documentation for any specific configuration changes."